Assets/Shopify/UIToolkit/ShopControllers/Editor/ShopControllerBaseEditor.cs

```diff
@@ -87,5 +79,53 @@ public abstract class ShopControllerBaseEditor : Editor, IShopControllerBaseEdit
             EditorGUILayout.PropertyField(serializedObject.FindProperty("_appleMerchantID"));
             #endif
         }
+
+        void IShopControllerBaseEditorView.DrawShopCredentialsVerifier() {
+            EditorGUILayout.Separator();
+
+            var disableCredentialsForm = _verifier.HasVerifiedCredentials() || _verifier.RequestInProgress;
+
+            EditorGUI.BeginDisabledGroup(disableCredentialsForm);
+
+            EditorGUI.BeginChangeCheck();
```

Comment:
This is interesting. You mentioned in slack about how the editor stuff is Immediate mode rendering and this API makes it pretty clear that it is. Good to now for future editor scripts!